include/vtzero/property.hpp

```diff
@@ -74,6 +74,11 @@ namespace vtzero {
 
     }; // class property
 
+    /// properties are equal if they contain the same key & value data.
+    inline constexpr bool operator==(const property & lhs, property & rhs) noexcept {
```

Comment:
Missing `const`. Please use formatting convention used in the rest of the code (no space before `&`).

Reply:
@joto this is copied from the operators [defined in property_value.hpp](https://github.com/mapbox/vtzero/blob/c59d9dd571ce9060bca704c429e60bd17c52ab09/include/vtzero/property_value.hpp#L233-L235), which seem to be missing const as well.

Reply:
nvm, I missed the second `const`!